gammapy/modeling/models/core.py

```diff
@@ -105,10 +105,20 @@ def copy(self):
         """A deep copy."""
         return copy.deepcopy(self)
 
-    def to_dict(self):
+    def to_dict(self, full_output=True):
```

Comment:
I think for writing we should change the default behaviour to `full_output= False`...

Reply:
If I change for full_output = False as default then I would not apply simplified output to frozen as it too important to be hidden by default.

Reply:
I think assuming a parameter is not frozen by default is fine...that's what we do in the `Parameter` class as well.

Reply:
ok